### DISCUSSION TITLE
Unexpected error

### DISCUSSION THREAD

**Discussion by KristofDB79** [wlcrs/huawei_solar#444] [Connecting to the inverter]
I'm trying to connect a SUN2000-4.6KTL-L1 inverter, but receiving error "Unexpected error"

Connecting on the LAN, not on the build in WLAN.   "Local O&M" has been enabled and working, as port 6607 is responding.

No fixed IP was set, due to settings were not accepted.

Over the build in WLAN connection is working, but I'm moving to another system and can't use the WLAN connection anymore.

Any help would be appreciated.

**Comment by KristofDB79** [comment]
Issue is solved

**Reply by alessandrocolos** [reply]
What's the solution?
I have the same problem now :(

Thanks
Alessandro

**Reply by KristofDB79** [reply]
There was still a connection active on the WLAN of the invertor to a previous HA on another device. (I was moving HA to new hardware)

A solution for not able to set a fixed IP i did not find.

Hope this help for the problem you encounter.

Regards,


Kristof
